src/utils/utils_log.c

```diff
@@ -254,6 +254,16 @@ void util_log_init(void) {
 
         memcpy(file, arg, len);
         file[len] = '\0';
+
+        // for security reasons check if we don't open a symlink file
+        if (util_is_symlink(file)) {
+            loggerConfig.output = stderr;
+            LOG_ERR("Output file %s exists and is a symlink - logging disabled",
+                    file);
+            loggerConfig.output = NULL;
+            return;
+        }
+
```

Comment:
We open for write only here - i'm pretty sure that is it not an security issue.

Reply:
according to security guidelines we should never open symlinks